report-viewer/src/model/factories/OverviewFactory.ts

```diff
@@ -83,13 +91,64 @@ export class OverviewFactory {
       matchSensitivity,
       dateOfExecution,
       duration,
-      metrics,
+      comparisons,
+      distributions,
       clusters,
       totalComparisons,
       new Map()
     )
   }
 
+  /**
+   * Gets the overview file based on the used mode (zip, local, single).
+   */
+  public static getOverview(): Overview {
```

Comment:
Also a quite complex method especially the construction of the Overview object. Something to rework in the future